scripts/monitor/requirements.txt

```diff
@@ -0,0 +1,7 @@
+bokeh
+pandas
+numpy
+matplotlib
+pandas-bokeh
+operator
+os
```

Comment:
I think `os` comes with python, so no need to specify that.
Where's `operatro` needed?
We try to keep the number of dependencies as low as possible, but keep the ones that make our lives easier.